data/mods/Xedra_Evolved/player/professions.json

```diff
@@ -453,6 +453,46 @@
       "female": { "entries": [ { "item": "sports_bra" }, { "item": "boxer_shorts" } ] }
     }
   },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "hobby_eater",
+    "name": "Awakened Eater",
+    "description": "Ever since the Cataclysm you've been able to clearly see elements of other realities that have intruded your own.  You also think you'd be able to consume these intrusions to give you power far beyond a normal human.",
+    "points": 4,
+    "spells": [ { "id": "spring_heeled_leap", "level": 4 }, { "id": "spell_dodge", "level": 4 } ],
+    "traits": [ "EATER" ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "hobby_dreamer",
+    "name": "Awakened Dreamer",
+    "description": "Ever since the Cataclysm you've become far more aware of the weakened gap between realities.  Specifically, of what is on the other side.  Surprisingly, it doesn't seem universally hostile, and you think you'd be able to bring some of these entities over if you desired.",
+    "points": 4,
+    "spells": [ { "id": "create_dream_dross", "level": 4 }, { "id": "summon_shifter", "level": 4 } ],
+    "traits": [ "DREAMER" ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "hobby_inventor",
+    "name": "Awakened Inventor",
+    "description": "Ever since the Cataclysm you've begun having crazy ideas on how to create technological devices.  While conventional logic would label these designs as silly or even insane, something tells you that they'd work regardless.",
```

Comment:
Do you want to give them some randomized recipes like I did in these eoc's for the professions?
```
{
    "type": "effect_on_condition",
    "id": "EOC_INVENTOR_START_RECIPES",
    "eoc_type": "EVENT",
    "required_event": "game_avatar_new",
    "condition": { "compare_string": [ "xe_inventor", { "context_val": "avatar_profession" } ] },
    "effect": [
      { "u_roll_remainder": [ "software_math_inventor", "software_electronics_reference_inventor" ], "type": "recipe" },
      { "u_roll_remainder": [ "inventor_leg_weight", "inventor_jump_boots" ], "type": "recipe" },
      { "u_roll_remainder": [ "inventor_shotgun", "inventor_pistol" ], "type": "recipe" }
    ]
  },
  {
    "type": "effect_on_condition",
    "id": "EOC_DREAMSMITH_START_RECIPES",
    "eoc_type": "EVENT",
    "required_event": "game_avatar_new",
    "condition": { "compare_string": [ "xe_dreamsmith", { "context_val": "avatar_profession" } ] },
    "effect": [
      { "u_roll_remainder": [ "forged_dreamstuff_ingot" ], "type": "recipe" },
      { "u_roll_remainder": [ "dreamdross_bo", "dreamdross_knife", "dreamdross_club" ], "type": "recipe" },
      { "u_roll_remainder": [ "dreamdross_lump", "forged_dreamstuff_ingot" ], "type": "recipe" }
    ]
  }
```

Reply:
I tried with both avatar_profession and avatar_custom_profession but it looks like those do not detect hobbies.  Hobby inventors and dreamsmiths will just have to start off a little slower I guess.